### ISSUE TITLE
getFetch(...) is not a function in Node.js SDK

### ISSUE THREAD

**Issue by laptransang** [splitio/javascript-client#505]
I'm using Node.js SDK and found this issue when building the project, it looks like it has an issue with the package node-fetch, I can fix it by a change that like of code, could I send the pull request to fix that issue.

https://github.com/splitio/javascript-client/blob/development/src/services/getFetch/node.js

```javascript
// from
let nodeFetch = require('node-fetch');

// to
let nodeFetch = require('node-fetch').default;

```

![image](https://user-images.githubusercontent.com/14858308/104898245-f33be300-59ab-11eb-8b68-253f8c223b7f.png)

**Comment by chillaq** [comment]
Thanks @laptransang will check the pull request with the team

**Comment by EmilianoSanchez** [comment]
Hi @laptransang,

Thanks for your collaboration.
The same [issue](https://github.com/node-fetch/node-fetch/issues/1037) has been reported recently by another user of the 'node-fetch' library without responses yet, but since we are importing it as the authors suggest, we will merge your PR and then apply a slight update in case it might impact other users:

```
let nodeFetch = require('node-fetch');

// Handle node-fetch issue https://github.com/node-fetch/node-fetch/issues/1037
if(typeof nodeFetch !== 'function') nodeFetch = nodeFetch.default;
```

We will apply a patch soon, probably will be out next week, will let you know.

**Comment by laptransang** [comment]
> Hi @laptransang,
> 
> Thanks for your collaboration.
> The same [issue](https://github.com/node-fetch/node-fetch/issues/1037) has been reported recently by another user of the 'node-fetch' library without responses yet, but since we are importing it as the authors suggest, we will merge your PR and then apply a slight update in case it might impact other users:
> 
> ```
> let nodeFetch = require('node-fetch');
> 
> // Handle node-fetch issue https://github.com/node-fetch/node-fetch/issues/1037
> if(typeof nodeFetch !== 'function') nodeFetch = nodeFetch.default;
> ```
> 
> We will apply a patch soon, probably will be out next week, will let you know.

Thanks for your update @EmilianoSanchez, it great if we have a condition like that to make sure it compatible with the other project.

**Comment by EmilianoSanchez** [comment]
Hi @laptransang , we have released JS SDK v10.15.3, which includes your suggestion for the node-fetch import issue.

I am closing out this issue, but if you try it and are still facing the issue, feel free to re-open it.

Thanks, 
Emiliano
